src/scripts/main.js

```diff
@@ -1,3 +1,27 @@
 'use strict';
 
-// write code here
+document.querySelectorAll('.tree li').forEach((li) => {
```

Comment:
Consider using a more specific selector if possible to avoid unnecessary DOM queries. This will improve performance, especially if the tree structure is large.